tracer/test/snapshots/Iast.StrictTransportSecurity.AspNetCore2.texthtml.500.invalidhttps.verified.txt

```diff
@@ -0,0 +1,56 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet_core.request,
+    Resource: GET /iast/stricttransportsecurity,
+    Service: Samples.Security.AspNetCore2,
+    Type: web,
+    Error: 1,
+    Tags: {
+      aspnet_core.route: iast/stricttransportsecurity,
+      component: aspnet_core,
+      env: integration_tests,
+      error.msg: The HTTP response has status code 500.,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.route: iast/stricttransportsecurity,
+      http.status_code: 500,
```

Comment:
I was surprised to see that this didn't raise a vulnerability. Just want to double check, when the response code is 500 we intend to skip the headers analyzer?

Reply:
Yes, there are some response codes that skip the vulnerability. The vulnerability is not launched when the return code of the request is 301, 302, 304, 307, 404, 410 or 500.